code/controllers/subsystem/rogue/devotion.dm

```diff
@@ -15,6 +15,7 @@
 #define CLERIC_REGEN_DEVOTEE 0.3
 #define CLERIC_REGEN_MINOR 0.5
 #define CLERIC_REGEN_MAJOR 0.8
+#define CLERIC_REGEN_ABSOLUTIONIST 5
```

Comment:
minor nitpick but this is pretty unneeded especially with how low the psydon miracle costs are.
Major regen is 80 across 10 seconds.
I get that you want to add stuff for flavor but a cosmetic trait works just as well, as this doesn't even have any real gameplay effects, nor is it particularly noticeable.